src/api/api.ts

```diff
@@ -23,6 +38,19 @@ export const register = async (userData: FormData) => {
         throw error.response.data;
     }
 };
+export const getCurrentUser = () => {
```

Comment:
this is not the right place to have this function, should be in a helper file that has common functions